web-client/src/app/pipes/xrp-date.pipe.ts

```diff
@@ -1,14 +1,15 @@
 import { Pipe, PipeTransform } from '@angular/core';
+import * as moment from 'moment';
 
 @Pipe({
   name: 'xrpDate',
 })
 export class XrpDatePipe implements PipeTransform {
   transform(date: number | undefined): string {
     if (date) {
-      const from = new Date('01/01/2000');
-      const finalDate = new Date(from.setSeconds(from.getSeconds() + date));
-      return new Date(finalDate).toLocaleString();
+      const from = moment.utc('01/01/2000', 'DD/MM/YYYY');
+      const finalDate = moment(from).add(date, 'seconds');
+      return finalDate.local().format('YYYY-MM-DD HH:mm:ss');
```

Comment:
Are you not supposed to do `local(true)`? 

as per the [doc](https://momentjs.com/docs/#/manipulating/local/)?